README.md

```diff
@@ -95,3 +95,9 @@ http {
     }
 }
 ```
+
+### Subresource support
+
```

Comment:
Could we explain the user-facing behavior first then follow with the technical explanation, e.g.,

nginx-sxg-module automatically includes signatures of subresources in its responses, allowing prefetching.

Reply:
nice suggestion.